src/chocolatey.console/Program.cs

```diff
@@ -291,23 +289,23 @@ private static void write_warning_for_incompatible_versions()
 command.");
         }
 
-        private static void check_installed_dotnetfx_version()
+        private static void ThrowIfNotDotNet48()
         {
-            if (Platform.get_platform() == PlatformType.Windows)
+            if (Platform.GetPlatform() == PlatformType.Windows)
             {
                 // https://learn.microsoft.com/en-us/dotnet/framework/migration-guide/how-to-determine-which-versions-are-installed#minimum-version
-                const int NET48RELEASEBUILD = 528040;
-                const string REGKEY = @"SOFTWARE\Microsoft\NET Framework Setup\NDP\v4\Full\";
+                const int net48ReleaseBuild = 528040;
+                const string regKey = @"SOFTWARE\Microsoft\NET Framework Setup\NDP\v4\Full\";

```

Comment:
We did... Do we want to maintain that for _local_ constants? Typically locals are all more or less the same casing (for us, usually that's `camelCase`) regardless of what they are. Happy to go through and change these, but didn't seem that useful or necessary at a glance.

Reply:
Most member variables that I saw in the codebase pretty consistently seem to use `_camelCase` with the underscore if they're private and otherwise PascalCase for protected/internal/public, so I don't think there's much danger of confusing a local variable for them.

IMO the fact it's a local is more important to convey than the fact it's a const. Its scope of use is much narrower than a class member is, and I think if we make any locals PascalCase it's liable to cause confusion as to where we'd expect them to be declared and where we can use them.

Reply:
That's a good point. From what I can tell the editorconfig file doesn't seem to have any standards that can be applied to local variables.

From what I can see in VS at least: 
![image](https://user-images.githubusercontent.com/32407840/229809157-ff951430-e4f5-43ac-9458-0dc94ea54f19.png)

Reply:
We would need to add something else on top of this to codify this I think.

Reply:
As discussed, this will be captured outside of this PR, so happy to move forward as-is for now.